packages/strapi-plugin-users-permissions/config/schema.graphql

```diff
@@ -20,9 +20,24 @@ module.exports = {
       description: String
       type: String
     }
+
+    input UsersPermissionsLoginInput {
+      identifier: String
+      password: String
+      provider: String = "local"
+    }
+
+    type UsersPermissionsLoginPayload {
+      jwt: String
+      user: UsersPermissionsUser
+    }
   `,
   query: `
     me: UsersPermissionsMe
+    login(input: UsersPermissionsLoginInput): UsersPermissionsLoginPayload
```

Comment:
Why is login a mutation though? I'm just curious.

Reply:
Because normally you mutating the the state. isLoggedIn from false to true. Currently the cms isn't mutating/saving any data, but I think this will be added in the future. For example for a multi device Login, where you can remotely logout your account from other browsers.